src/main/java/de/tum/in/www1/artemis/service/ConflictingResultService.java

```diff
@@ -0,0 +1,84 @@
+package de.tum.in.www1.artemis.service;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import de.tum.in.www1.artemis.domain.Feedback;
+import de.tum.in.www1.artemis.domain.Result;
+import de.tum.in.www1.artemis.domain.modeling.ConflictingResult;
+import de.tum.in.www1.artemis.domain.modeling.ModelAssessmentConflict;
+import de.tum.in.www1.artemis.repository.ConflictingResultRepository;
+
+@Service
+public class ConflictingResultService {
```

Comment:
This service and the methods within it should get java docs.
E.g. what is a double conflicting result?
Also there is commented out code and a TODO where I wonder if it should be fixed now or removed.

Reply:
Well the double conflicting result filter has been made obsolete by a previous fix.. so thanks for pointing that one out so i could remove it!^^ Also just added javadocs to the service methods and removed the todo